src/main/java/com/google/sps/servlets/TestUploadServlet.java

```diff
@@ -0,0 +1,178 @@
+    
```

Comment:
I guess you can remove this file, we have `databaseInit.java` that is the same so you can update it.